vendor/vendor.json

```diff
@@ -219,21 +219,15 @@
 			"revision": "e7df0c1118c15c0b35fe08f183ca084269ea6542",
 			"revisionTime": "2020-02-13T16:48:48Z"
 		},
-		{
-			"checksumSHA1": "tPUy5ckuRcNSroe+kj8PH/edRuo=",
```

Comment:
why these dependencies are changing?

Reply:
my mistake. should not have sent that file. But it's because when you run "go sync" it randomly changes the order of the dependencies (not the dependencies themselves, just the order in the file)